tools/benchmark/post_run_checks.py

```diff
@@ -7,6 +7,10 @@ def main():
     max_unaccounted = 200 * 1024 * 1024  # 200mb
 
     client = redis.Redis(decode_responses=True)
+    info = client.info("server")
+    # Check version upgrade finsihed from last released version to last weekly docker build
+    assert info["dragonfly_version"] == "df-HEAD-HASH-NOTFOUND"
```

Comment:
So, `post_run_checks.py` only runs after upgrade? or just runs at the end and we reach end only after an upgrade. 🤔

Reply:
It runs at the end when memtier finish running and also the upgrade was done